tools/codegen/cmd/mixgenproc/BUILD

```diff
@@ -0,0 +1,14 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
I'm fine with moving it once things have settled, but generally agree that having all the cmds in one place makes sense.